### PR TITLE
Bug fix for User Guide

### LINKED ISSUE #407: [PE-D] File is unreadable
Unable to look back at what I stored in the file. Files give weird symbols.

-------------
Labels: `type.FeatureFlaw` `severity.Medium`
original: FaliciaOng/ped#1

### PR DESCRIPTION
**Pull request by ivanaitzliddat**
Fixes #407 , #412 , #413 , #416 , #419 , #420 , #426

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -207,10 +224,17 @@ Edits an existing session so that you can change the details of a session.<br>
 >- There are 3 editable fields: _session name_, _session date_ and the _list of participants_ in the session.
 >  - At least 1 field has to be edited for the command to run.
 >  - More than 1 field can be edited in a single run of the command.
->- When editing the _list of participants_, existing participants must be included in the command.
->  - Example: If the session previously created had Alice and Bob with session unique identifier of 1 
->    and you wish to edit it to include Charlie, a valid edit command would be
-     `session /edit /sid 1 /pl Alice Bob Charlie`.
+>- When editing the _list of participants_, existing participants, including the group that was declared when
+>  creating the session, has to be included.
```
[the remark this answers is not in the document]
Comment (matheril) [reply docs/UserGuide.md:228]: It might be slightly misleading to say "including the group that was declared..." as the edit command does not take in a group name as an argument. Perhaps rephrase this to "including those part of the group declared when creating the session must be included."

File: docs/UserGuide.md [hunk 2]
```diff
@@ -308,17 +332,22 @@ There are 2 ways that you can create an activity:
...
+>- The values in `[TOTAL_COST]` and `[COST1 COST2 ...]` are decimal values with a maximum of 12 digits before
+>  and 2 digits after the decimal point, if any.
+>- The values in `[GST_PERCENTAGE]` and `[SERVICE_CHARGE]` are decimal values that range from 0 to 100,
+>  with a maximum of 3 digits before and 2 after the decimal point, if any.
+>- If the payer is also involved in the activity, the payer's name has to be included in the list of participants
+>  - Example: Alice paid for a movie which she watched with Bob in a session with a session unique identifier of 1,
+>    costing a total of $20. The correct command format is: <br>
+>    `activity /create /sid 1 /n movie /p Alice /i Alice Bob /co 20`
+>- Given that the values can only be displayed up to 2 decimal places, rounding errors may seem to have occurred
+>  in the calculations.
```
Comment (matheril) [inline docs/UserGuide.md:350]: The phrase "may seem to have occurred" seems vague and passive. Consider replacing it with "may occur". Or rephrase the sentence to:

"All values displayed are rounded off to 2 decimal places." The phrase "rounded off" already implies that inaccuracies may result, but you could modify this to "All values displayed are rounded off to 2 decimal places. This may result in slight inaccuracies." to reiterate this to the user.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -508,8 +540,9 @@ must pay and to whom they should pay for all debts to be resolved.<br>
 <hr>
 
 ## _Group Management_
-> A group represents one or more individuals. Several commands use groups to quickly identify a group of individuals
-  without having to manually enter their details one by one.
+> A group represents one or more individuals. The only benefit of a group is to quickly identify a group of individuals
```
Comment (matheril) [inline docs/UserGuide.md:543]: Consider changing "benefit" to "purpose" to make the language seem more objective.

File: docs/UserGuide.md [hunk 4]
```diff
@@ -508,8 +540,9 @@ must pay and to whom they should pay for all debts to be resolved.<br>
...
+> without having to manually enter their details one by one when creating a session.
+> This is mainly to improve user experience when using SplitLah.
```
Comment (matheril) [inline docs/UserGuide.md:545]: This seems vague. "improve user experience" is an umbrella term, so consider narrowing this down to the specific part of the user experience that is improved. Consider:
"Groups help you spend less time adding the same people to multiple sessions". Alternatively, you could just remove this line.

**Review by matheril — APPROVED** [review]
Some language changes suggested, but they are minor. Looks good to merge.

**Review by warrencxw — CHANGES_REQUESTED** [review]
Only added a single change as Saurav has gave suggestions to most of the places that have be changed.

**Review by warrencxw — APPROVED** [review]
Looks good to merge.

**Review by froststein — APPROVED** [review]
Looks good to merge
